### ISSUE TITLE
Jnet Freeze Mid-Game - No Interaction

### ISSUE THREAD

**Issue by PyreflY7** [mtgred/netrunner#3734]
My jinteki.net games at some point just stop letting me press buttons or type anything. Anything I enter into chat doesn't go anywhere, I can't activate or use any cards and my opponent is left wondering if I have just walked away. 

I had a look at previous posts on this kind of issue, but they seem to be based on a particular card or interaction (from what I could see). My issue seems very random. It might be something to do with a "debug script or continue" prompt that comes up when I get on the website. But it has previously run fine.

This is particularly an issue as I have an international league coming up and I would like to avoid being DQ-ed from every single round. Any help would be welcome!

**Comment by Saintis** [comment]
What browser are you using?

Also, next time it happens, can you check if there are any messages in the Developer Console. That might give some clues to why it happens.

Have you also tried disconnecting and reconnecting to the game to see if that helps?

**Comment by PyreflY7** [comment]
@Saintis I am using Firefox.

What's the Developer Console? 

I did try disconnecting and rejoining. I just got a message saying that the connection was aborted.

**Comment by Saintis** [comment]
It's a window that will give you details about the javascript running the webpage. Details here: https://developer.mozilla.org/en-US/docs/Tools/Web_Console/Opening_the_Web_Console

If you can take a screenshot of the web console next time your game hangs we can see if there are a lot of errors appearing.

If you could also tell us more about this "debug script or continue" prompt you see that might give us more clues.

**Comment by PyreflY7** [comment]
@Saintis I have screenshotted some instances below

What happens when I first get onto Jnet:
https://user-images.githubusercontent.com/40968261/42624611-a9c81c6c-85bd-11e8-83b2-0c667f9fe0b5.png

What happens when I click debug script:
https://user-images.githubusercontent.com/40968261/42624689-ea658a52-85bd-11e8-863e-e6920a757ad9.png

I normally click continue a few times and it runs after that. I did just complete a game without issue. But I'll post again if there are any issues.
